### ISSUE TITLE
One IP, multiple domains and ROOT_URL

### ISSUE THREAD

**Issue by wujekbogdan** [arunoda/meteor-up-legacy#657]
Hi,
- I have multiple domains bound to the same IP address. 
- Meteor is running at port 80.
- Apache is running at port 8080.
- in the mup.json file, ROOT_URL is set to X.com

But when I enter Y.com URL in browser I can still access Meteor application. Shouldn't it be acessible only on the X.com domain? In Apache I can accomplish it using virtualhosts. Is it possible to the same using mup?

**Comment by devdudeio** [comment]
https://github.com/arunoda/meteor-up/wiki/Using-Meteor-Up-with-NginX-vhosts

you can define vhosts for nginx (instead of apache2) in /etc/nginx/sites-available/YOUR_CONFIG

**Comment by wujekbogdan** [comment]
Yeah, I've seen it, but It's not a solution but a workaround. It forces me to use nginx instead of apache.

//EDIT:
I've read it once again, carefully this time. I can use both web servers: apache and nginx. nginx runs at 80 port only as a reverse proxy for other services. In this case as a proxy for meteor and apache running on differnt ports (lets say at 3000 and 8080).

We can close this issue, because it's not meteor-up related, but server configuration issue.
